app/frontend/components/domains/permit-application/review-permit-application-screen.tsx

```diff
@@ -69,78 +85,131 @@ export const ReviewPermitApplicationScreen = observer(() => {
       onReferenceNumberChange(referenceNumberSnapshot)
     }
   })
-
+  const permitHeaderHeight = document.getElementById("permitHeader")?.offsetHeight
```

Comment:
default to using `refs` when possible instead of DOM api in React. In this case the `permitHeader` is directly available below, so use a `ref` to access the DOM node https://react.dev/learn/manipulating-the-dom-with-refs#best-practices-for-dom-manipulation-with-refs